ingenialink/eoe/network.py

```diff
@@ -129,43 +134,44 @@ def connect_to_slave(  # type: ignore [override]
             connection_timeout,
             servo_status_listener,
             net_status_listener,
-            True,
+            is_eoe=True,
         )
 
     def __wait_eoe_starts(self) -> None:
-        """Wait until the EoE service starts the EoE or the timeout was reached"""
+        """Wait until the EoE service starts the EoE or the timeout was reached."""
         status = self._get_status_eoe_service()
         time_start = time.time()
-        while not status & self.STATUS_EOE_BIT and self.WAIT_EOE_TIMEOUT > time.time() - time_start:
+        while not status & self.STATUS_EOE_BIT and time.time() - time_start < self.WAIT_EOE_TIMEOUT:
             time.sleep(0.1)
             status = self._get_status_eoe_service()
         if not status & self.STATUS_EOE_BIT:
             logger.warning("Service did not starts the EoE")
 
     def __reconfigure_drives(self) -> None:
-        """Reconfigure all the slaves saved in the network"""
-        for ip_addr, slave_id in self._configured_slaves.items():
-            try:
+        """Reconfigure all the slaves saved in the network."""
+        try:
+            for ip_addr, slave_id in self._configured_slaves.items():
                 self._configure_slave(slave_id, ip_addr)
-            except ILError as e:
-                logger.error(e)
+        except ILError:
+            logger.exception("Error reconfiguring drives")
 
-    def disconnect_from_slave(self, servo: EthernetServo) -> None:  # type: ignore [override]
+    def disconnect_from_slave(self, servo: EthernetServo) -> None:  # type: ignore [override] # noqa: D102
```

Comment:
Doesn't this function inherit the parent class's docstring?

Reply:
Fixed by adding the override decorator.